### ISSUE TITLE
최종 성능 정리

### ISSUE THREAD

**Issue by februa22** [nlp-research/biaffineparser#77]

**Comment by februa22** [comment]
경진대회 테스트셋 파싱 결과 파일: 
[all.pos.utf8.json.test.txt](https://github.com/nlp-research/biaffineparser/files/2382216/all.pos.utf8.json.test.txt)

**Comment by februa22** [comment]
Test accuracy:

No. | Model | UAS | LAS
--- | --- | :---: | :---:
1 | Biaffine (Baseline) | 91.77 | 89.37
2 | Biaffine + arc attention with label head (sum) | 91.82 | 89.34
3 | Biaffine + arc attention with label head (weight sum) | 91.88 | 89.52
4 | Biaffine + arc attention with label head and dep (weight sum) | 91.93 | 89.53

**Comment by februa22** [comment]
iss73-5

```json
{
    "num_lstm_units": 400,
    "num_lstm_layers": 4,
    "arc_mlp_units": 600,
    "label_mlp_units": 200,
    "word_embed_size": 200,
    "char_embed_size": 200,
    "pos_embed_size": 100,
    "optimizer": "adam",
    "learning_rate": 0.001,
    "decay_factor": 0.9,
    "num_train_epochs": 50,
    "train_filename": "data/sejong.char.train.csv",
    "dev_filename": "data/sejong.char.test.csv",
    "out_dir": "parser/model_iss73-5",
    "word_vocab_name": "word.pkl",
    "char_vocab_name": "char.pkl",
    "pos_vocab_name": "pos.pkl",
    "rel_vocab_name": "rel.pkl",
    "head_vocab_name": "head.pkl",
    "word_embed_file": "embeddings/words.pos.original.vec",
    "char_embed_file": "",
    "pos_embed_file": "",
    "word_embed_matrix_file": "embeddings/word_embed_matrix.txt",
    "char_embed_matrix_file": "embeddings/char_embed_matrix.txt",
    "pos_embed_matrix_file": "embeddings/pos_embed_matrix.txt",
    "embed_dropout": 0.33,
    "lstm_dropout": 0.33,
    "mlp_dropout": 0.33,
    "batch_size": 128,
    "device": "gpu",
    "debug": false,
    "inference_input_file": "",
    "inference_output_file": "parser/model_iss73-5/sejong.char.train.inference.tsv"
}
```
